### DISCUSSION TITLE
Chroma DB

### DISCUSSION THREAD

**Discussion by MarkJoel60** [griptape-ai/griptape#178] [Ideas]
What would it take to develop a driver for ChromaDB? Is this something that can be done by a relative novice, or does the driver require someone with deep system knowledge of ChromaDB

**Comment by vasinov** [comment]
@MarkJoel60, should be pretty easy to implement as a vector store driver. Check out how the [Pinecone integration](https://github.com/griptape-ai/griptape/blob/main/griptape/drivers/vector/pinecone_vector_store_driver.py) is implemented.
